multiple-cursors-core.el

```diff
@@ -442,6 +442,16 @@ you should disable multiple-cursors-mode."
       (multiple-cursors-mode 0)
     (deactivate-mark)))
 
+(defun mc/repeat-command ()
```

Comment:
Could this function take in a numerical-arg to decide which command to run from the history. When called interactively it could use `completing-read` to complete the command and then get its position from `command-history`. In this case I think we could also get rid of the `y-or-n-p` because the user will be able to `C-g` out of the selection.

Reply:
Doesn't this invalidate the meaning of `mc/repeat-command`? Its purpose is to clone effect of `M-x` into fake cursors. Example; `M-x` `insert-char` `1`. _Inserted into fist line_. Then `C-:`. _Inserted into other lines_.

For a simple repeat action, `C-x M-:` then `C-:` should be enough.

Reply:
Awesome, I definitely like the option to toggle `y-or-n-p`. Thanks for all of this!